_pages/basics_report-on-demand.md

```diff
@@ -955,19 +973,23 @@ Execute the following request: **http://localhost:33333/dynamicjasper/report**.
 
 ## Get preferences
 
-Execute the following request: **http://localhost:33333/preferences/preferences?entity={{entity}}&service={{service}}**.
+Execute the following request: **http://localhost:33333/preferences/preferences?entity={{entity}}&service={{service}}&type=REPORT**.
 
 | Element | Meaning |
 |--|--|
 | localhost:33333 | Indicates the host |
 | /preferences | Indicates the service to be queried |
 | /preferences | Indicates the method of the service that is going to be executed |
-| {{entity}} | Indicates the entity to filter the preferences |
-| {{service}} | Indicates the service to filter the preferences |
+| ?entity=**ENTITY** | Indicates the entity to filter the preferences |
```

Comment:
| ?entity=**ENTITY** | Indicates the entity to filter the preferences |

changes by

| ?entity=**ENTITY** | **ENTITY** indicates the entity to filter the preferences |